### PR TITLE
Updated cost of PvP-Items

### PR DESCRIPTION
**Pull request by Blackwidow-sudo**
### Updates for the PvP-Collection
- Update cost of S1 items
- Add cost of S2 nonset- & set-items
- Minor localisation fix
- ~~Added alliance- and horde-talisman (pvp)~~

### CODE REVIEW

**Comment by Blackwidow-sudo** [comment]
I will also update the costs of S1 Setitems, even though you cant see them in the dropdown.

_**More commits coming today**_

**Comment by Blackwidow-sudo** [comment]
I think i got everything. I will look over it tonight, but its ready to merge.

File: AtlasLootClassic/Data/VendorPrice.lua [hunk 1]
```diff
@@ -1287,7 +1303,7 @@ local VENDOR_PRICES = {
     [15198] = "pvpAlterac:20:pvpArathi:20:pvpWarsong:20",
     [15197] = "pvpArathi:3:pvpWarsong:3",
     [15196] = "pvpArathi:3:pvpWarsong:3",
-    [12584] = "honor:19125:pvpArathi:20",
+    [12584] = "honor:9563:pvpArathi:20",
```
Comment (Tenies) [inline AtlasLootClassic/Data/VendorPrice.lua:1306]: [32807] = "honor:17850:pvpArathi:40",
    [32799] = "honor:17850:pvpArathi:40",
    [32979] = "honor:17850:pvpArathi:40",
    [32798] = "honor:17850:pvpArathi:40",
    [32800] = "honor:17850:pvpArathi:40",
    [32802] = "honor:17850:pvpArathi:40",
    [32808] = "honor:17850:pvpArathi:40",
    [32998] = "honor:17850:pvpArathi:40",
    [32803] = "honor:17850:pvpArathi:40",
    [32804] = "honor:17850:pvpArathi:40",
    [32797] = "honor:17850:pvpArathi:40",
    [32806] = "honor:17850:pvpArathi:40",
    [32988] = "honor:17850:pvpArathi:40",
    [32805] = "honor:17850:pvpArathi:40",
    [32801] = "honor:17850:pvpArathi:40",
    [32795] = "honor:17850:pvpEye:40",
    [32787] = "honor:17850:pvpEye:40",
    [32981] = "honor:17850:pvpEye:40",
    [32786] = "honor:17850:pvpEye:40",
    [32788] = "honor:17850:pvpEye:40",
    [32790] = "honor:17850:pvpEye:40",
    [32796] = "honor:17850:pvpEye:40",
    [32999] = "honor:17850:pvpEye:40",
    [32791] = "honor:17850:pvpEye:40",
    [32792] = "honor:17850:pvpEye:40",
    [32785] = "honor:17850:pvpEye:40",
    [32794] = "honor:17850:pvpEye:40",
    [32990] = "honor:17850:pvpEye:40",
    [32793] = "honor:17850:pvpEye:40",
    [32789] = "honor:17850:pvpEye:40",
    [32820] = "honor:11794:pvpWarsong:20",
    [32811] = "honor:11794:pvpWarsong:20",
    [32980] = "honor:11794:pvpWarsong:20",
    [32810] = "honor:11794:pvpWarsong:20",
    [32814] = "honor:11794:pvpWarsong:20",
    [32812] = "honor:11794:pvpWarsong:20",
    [32821] = "honor:11794:pvpWarsong:20",
    [32997] = "honor:11794:pvpWarsong:20",
    [32816] = "honor:11794:pvpWarsong:20",
    [32817] = "honor:11794:pvpWarsong:20",
    [32809] = "honor:11794:pvpWarsong:20",
    [32819] = "honor:11794:pvpWarsong:20",
    [32989] = "honor:11794:pvpWarsong:20",
    [32818] = "honor:11794:pvpWarsong:20",
    [32813] = "honor:11794:pvpWarsong:20",
    [31984] = "arena:3750",
    [31966] = "arena:3750",
    [32025] = "arena:3750",
    [32055] = "arena:3750",
    [31959] = "arena:3750",
    [32014] = "arena:3750",
    [31986] = "arena:3750",
    [32044] = "arena:2625",
    [32026] = "arena:2625",
    [32052] = "arena:2625",
    [31965] = "arena:2625",
    [32028] = "arena:2625",
    [32053] = "arena:3150",
    [32963] = "arena:3150",
    [32964] = "arena:3150",
    [32046] = "arena:1125",
    [31958] = "arena:1125",
    [32027] = "arena:1125",
    [31985] = "arena:1125",
    [32003] = "arena:1125",
    [31978] = "arena:1125",
    [32961] = "arena:1125",
    [32045] = "arena:1875",
    [33309] = "arena:1875",
    [33313] = "arena:1875",
    [32962] = "arena:1000",
    [32054] = "arena:1000",
    [33943] = "arena:1000",
    [33946] = "arena:1000",
    [33076] = "arena:1000",
    [33077] = "arena:1000",
    [33937] = "arena:1000",
    [33949] = "arena:1000",
    [33078] = "arena:1000",
    [33952] = "arena:1000",
    [33940] = "arena:1000",

Comment (Tenies) [reply AtlasLootClassic/Data/VendorPrice.lua:1306]: Rest of s2 items, not including sets.

Comment (Blackwidow-sudo) [reply AtlasLootClassic/Data/VendorPrice.lua:1387]: Done👍

**Comment by Blackwidow-sudo** [comment]
**_S2 Sets incoming_**

**Comment by Blackwidow-sudo** [comment]
✔ Done

**Comment by Blackwidow-sudo** [comment]
✔Done

**Comment by Blackwidow-sudo** [comment]
oh wow 👀 @Tenies
how should i interpret this?
49fc1b83470bfe4780c193ecc9e19f2a7fe1f78b + 84afae5d10596bf1316a6d3435711f8bb0ef3ee4 + 561f28048e1b04f25da53e95870c72726feabc7e = ff42f759ef61459a7e408fec2641c0c7db97cb3e
Your commit msg was `Added Reins of the Dark War Talbuk price` but there are 380 changes.
There are clearly changes that were copied from my commits bcs even my whitespace, that i forgot to remove, is in it.
Was it a copy for yourself and you forgot to delete it?
Or were you trying to sneak my commits in as yours?
Its kinda sussy 🤔

**Comment by Tenies** [comment]
I totaly fcked up bro! When u made some of those changes i added them to the atlasloot i use so when i commited "Reins of the Dark War Talbuk price" it also took your chances. I'm so sorry about this bro! Didn't noticed it until now, i feel so bad! :(

**Comment by Tenies** [comment]
Maybe if we ask Hoizame to reverse this commit (ff42f75) and push it again?

**Comment by Blackwidow-sudo** [comment]
> Maybe if we ask Hoizame to reverse this commit ([ff42f75](https://github.com/Hoizame/AtlasLootClassic/commit/ff42f759ef61459a7e408fec2641c0c7db97cb3e)) and push it again?

nah its ok^^
I just wanted to know what happend here bcs it was very sussy and we all know that there are many guys out there that steal code.

Its on master and already published. Reverting, just so that i can see my name on it, would be very selfish and messy^^
Shit happens 😀

**Comment by Blackwidow-sudo** [comment]
@Hoizame if there is something wrong please tell me.
S1 Item costs are halved since S2. So i dont understand why this doesn't get merged 😮

**Comment by sintaxx** [comment]
How do I see the prices of "sets" pieces ?

**Comment by Blackwidow-sudo** [comment]
> How do I see the prices of "sets" pieces ?

You mean ingame?
I dont think you can see them.

**Comment by sintaxx** [comment]
Is there any reason why the prices for set pieces are not in AtlasLoot ?

**Comment by Blackwidow-sudo** [comment]
> Is there any reason why the prices for set pieces are not in AtlasLoot ?

well the prices are in atlasloot, you just cant see them because of the way sets are displayed.
When you click on a set you get a overlay with all the set-items. But currently there seems to be no way of displaying prices in that overlay.

**Pull request closed without merging**